### ISSUE TITLE
Char Location possible fix? 

### ISSUE THREAD

**Issue by Cat-Harkness** [3vi1/IMP#59]
That information is in the current log under \Gamelogs\....

[ 2018.04.10 23:08:44 ] (None) Jumping from ZD1-Z2 to 6-IAFR
[ 2018.04.10 23:09:53 ] (None) Jumping from 6-IAFR to Y-ORBJ
[ 2018.04.10 23:11:00 ] (None) Jumping from Y-ORBJ to UALX-3

**Comment by Cat-Harkness** [comment]
But it looks like IMP will not be able to tell what system you are in, until you undock, dock or jump

[ 2018.04.10 22:49:24 ] (None) Undocking from ZD1-Z2 X - Glacial Court to ZD1-Z2 solar system.

**Comment by 3vi1** [comment]
Yes, I've already put this level of fix into IMP, but haven't released it yet.  It works a little better in IMP because Imp will parse the older logs and set your position if you've changed systems in the last couple of days.

I'll go ahead and upload the latest version tonight or tomorrow - just as soon as I get time to play and test it more thoroughly.

**Comment by 3vi1** [comment]
I did not get to play/test last night, but I've gone ahead and built/uploaded a pre-release that fixes the character location:  https://github.com/3vi1/IMP/releases/tag/v0.9.7.7

Hopefully I'll get time tonight to test proper and wrap up any remaining issues.

**Comment by tt0h** [comment]
For the most part the fix works well.  The only issue I'm currently seeing is IMP doesn't know where you are after the use of a jump bridge - until you change systems via a gate.

**Comment by 3vi1** [comment]
Thanks for the info.  Being run out of Provi, I've not had a jump bridge to test with.  :)  I'll look into what it would take to move location tracking to the ESI interface this weekend to completely resolve this.
